src/components/JobActionsPopover.vue

```diff
@@ -89,13 +93,20 @@ export default defineComponent({
         await this.store.dispatch('user/updatePinnedJobs', { pinnedJobs: [...pinnedJobs] });
       }
       this.closePopover();
+    },
+    async runJobNow(job: any) {
+      if(job) {
+        await this.store.dispatch('job/runServiceNow', job)
```

Comment:
I think we should handle setting productStoreId from job instead of state value

Reply:
Yes sir. Understood as we discussed. Fixing it

Reply:
Check if we need to handle for shopifyConfigId as well

Reply:
Sir shopifyConfigId never comes from job, thus I guess it is okay to always set it from userState